lxd/storage/drivers/driver_zfs.go

```diff
@@ -34,7 +34,7 @@ var zfsDefaultSettings = map[string]string{
 	"setuid":     "on",
 	"exec":       "on",
 	"devices":    "on",
-	"acltype":    "posixacl",
+	"acltype":    "posix", // acltype "posix" is equivalent to "posixacl". Set as "posix" to avoid setting it multiple times on ensureInitialDatasets.
```

Comment:
Is `posix` newer than `posixacl` and if so, when did it get introduced?

Reply:
`posixacl` is older but was kept as an alias when `posix` was added in September 16th, 2020

Reply:
which version was it added in?

Reply:
Can you check this doesn't break LXD 5.21/stable with this binary sideloaded on a ZFS 0.8 system (bionic with hwe kernel) please.

Reply:
Unfortunately it breaks. As can be seen [here](https://github.com/openzfs/zfs/blob/zfs-0.8.3/module/zcommon/zfs_prop.c#L157-L163) support for `posix` wasn't added until later. I am pushing a new version that solves this problem.